### PR TITLE
Adds a lollipop synthesizer to mediborgs

### PR DESCRIPTION
**Pull request by ArchSunder**
Your patient was very brave, give them a lollipop

:cl:
 * rscadd: Mediborgs now have a lollipop dispenser.

### CODE REVIEW

**Comment by cloutcode** [comment]
idea: emagged lollipops contain neurotoxin

**Comment by jknpj** [comment]
It's missing the toxin type, unless you want it to have shitter juice on them

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -272,6 +272,7 @@
 	modules += new /obj/item/weapon/revivalprod(src)
 	modules += new /obj/item/weapon/inflatable_dispenser/robot(src)
 	modules += new /obj/item/robot_rack/bed(src)
+	modules += new /obj/item/weapon/cookiesynth/lollipop(src)
```
Comment (jknpj) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:275]: might want to add it to syndicate mediborgs too

Comment (ArchSunder) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:275]: poop

Comment (ArchSunder) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:275]: okay done

**Comment by SonixApache** [comment]
should make a subtype of lollipop with a bit of tricord in it for this
and then add a bag of those to latejoin CMO rewards
🙂

**Comment by ArchSunder** [comment]
Is it OK to try adding chems/emag toxins to something in this PR or should I make a new one with those changes

**Comment by jknpj** [comment]
Eh, just go for it.

**Comment by ArchSunder** [comment]
I copypasted a new type of lollipop containing tricord and gave it to the borgs. I can't get emagging to shit out toxin lollipops, there's probably something about the child synth inheriting functions or something else that I don't have knowledge of. But the medipops are tested and working

**Comment by ShiftyRail** [comment]
Code looks ok, huge thumbs up, what could go wrong?

**Comment by jknpj** [comment]
>what could go wrong?

Nothing but the huge chunk of copypasted lollipop code(that we already told them to avoid and use child types instead) and the lack of snowflake toxin that they didn't knew how to add but can easily be added through the `toxin_type` var

**Comment by ArchSunder** [comment]
> > what could go wrong?
> 
> Nothing but the huge chunk of copypasted lollipop code(that we already told them to avoid and use child types instead) and the lack of snowflake toxin that they didn't knew how to add but can easily be added through the `toxin_type` var

I did try that, the lollipops made with the emagged synth didn't contain the toxin.

**Comment by jknpj** [comment]
Maybe their reagent container was full?

**Comment by ArchSunder** [comment]
That was probably it, it looks like sonix gave them a small container so they couldn't hold much poison

**Comment by SonixApache** [comment]
copypaste aside, the lollipops can hold up to 15u of stuff - this code tries to add 5 more than it can even contain
if you want a poison pop, make a subtype of the normal lollipop that contains poison, and maybe raise the volume if you want it to be worthwhile

**Pull request merged**